### PR TITLE
babe: log block and slot number on verification

### PR DESCRIPTION
**Pull request by andresilva**

### CODE REVIEW

File: client/consensus/babe/src/verification.rs [hunk 1]
```diff
@@ -95,7 +95,10 @@ pub(super) fn check_header<B: BlockT + Sized>(
 
 	match &pre_digest {
 		PreDigest::Primary(primary) => {
-			debug!(target: "babe", "Verifying Primary block");
+			debug!(target: "babe",
+				"Verifying primary block #{} at slot: {}",
+				header.number(), primary.slot_number,
```
Comment (bkchr) [inline client/consensus/babe/src/verification.rs:100]:
```suggestion
				"Verifying primary block #{} at slot: {}",
				header.number(),
				primary.slot_number,
```

File: client/consensus/babe/src/verification.rs [hunk 2]
```diff
@@ -106,7 +109,11 @@ pub(super) fn check_header<B: BlockT + Sized>(
 			)?;
 		},
 		PreDigest::SecondaryPlain(secondary) if epoch.config.allowed_slots.is_secondary_plain_slots_allowed() => {
-			debug!(target: "babe", "Verifying Secondary plain block");
+			debug!(target: "babe",
+				"Verifying secondary plain block #{} at slot: {}",
+				header.number(), secondary.slot_number,
```
Comment (bkchr) [inline client/consensus/babe/src/verification.rs:114]:
```suggestion
				header.number(),
				secondary.slot_number,
```

File: client/consensus/babe/src/verification.rs [hunk 3]
```diff
@@ -115,7 +122,11 @@ pub(super) fn check_header<B: BlockT + Sized>(
 			)?;
 		},
 		PreDigest::SecondaryVRF(secondary) if epoch.config.allowed_slots.is_secondary_vrf_slots_allowed() => {
-			debug!(target: "babe", "Verifying Secondary VRF block");
+			debug!(target: "babe",
+				"Verifying secondary VRF block #{} at slot: {}",
+				header.number(), secondary.slot_number,
```
Comment (bkchr) [inline client/consensus/babe/src/verification.rs:127]:
```suggestion
				header.number(),
				secondary.slot_number,
```

**Review by bkchr — APPROVED** [review]

File: client/consensus/babe/src/verification.rs (hunk 1 above)
Comment (andresilva) [reply client/consensus/babe/src/verification.rs]: rustfmt formats println statements like I did 🙈

**Review by octol — APPROVED** [review]

**Pull request merged**